### PR TITLE
[oracle] shade dependencies in oracle-cdc connector jar #519

### PR DESCRIPTION
**Pull request by luoyuxia**
verified with shaded oracle-cdc connector manually.

### CODE REVIEW

File: flink-sql-connector-oracle-cdc/pom.xml [hunk 1]
```diff
@@ -52,7 +52,19 @@ under the License.
...
-                                    <include>*:*</include>
+                                    <include>io.debezium:debezium-api</include>
+                                    <include>io.debezium:debezium-embedded</include>
+                                    <include>io.debezium:debezium-core</include>
+                                    <incldue>io.debezium:debezium-ddl-parser</incldue>
+                                    <include>io.debezium:debezium-connector-oracle</include>
+                                    <include>com.ververica:flink-connector-debezium</include>
+                                    <include>com.ververica:flink-connector-oracle-cdc</include>
+                                    <include>org.antlr:antlr4-runtime</include>
+                                    <include>com.github.jsqlparser:jsqlparser</include>
```
Comment (leonardBang) [inline flink-sql-connector-oracle-cdc/pom.xml:63]: Also shade this dependency ?

**Review by leonardBang — COMMENTED** [review]
Thanks @luoyuxia for the contribution, I only left one minor comment.
Could you post your end to end test result here due to we don't have the e2e tests? Which should use Flink-1.13.x cluster and `oracle-cdc` sql jar with 2.1-SNAPSHOT version.

**Comment by luoyuxia** [comment]
截屏2021-11-03 下午3 47 39
截屏2021-11-03 下午3 47 57

Also provide the shaded jar for others, and checked by others. So, I thinks the shaded jar is ok.

**Review by leonardBang — APPROVED** [review]
LGTM

**Pull request merged**